roles/cs.aws-ses/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+ses_sns_topic_name_bounce: ses-bounce
+ses_sns_topic_name_complaints: ses-complaints
+ses_email:
```

Comment:
I would comment this out so it fails if the variable is not set instead of using empty value.